common/types/block.go

```diff
@@ -34,12 +35,33 @@ func (id BlockID) AsHash32() Hash32 {
 	return Hash20(id).ToHash32()
 }
 
+var layersPerEpoch int32 = 0
+
+// EffectiveGenesis marks when actual blocks would start being crated in the network, this will take account the first
+// genesis epoch and the following epoch in which ATXs are published
+var EffectiveGenesis int32 = 0
+
+func getLayersPerEpoch() int32 {
+	return atomic.LoadInt32(&layersPerEpoch)
```

Comment:
probably won't change after initial setting so I see no reason to use atomic.

Reply:
to settle race tests